hngi/SuperNigerians
# Feat/user update profile modal

**What does this PR do?**
This PR aims to add a modal tho the application to be used by a user to update his/her profile

> Summarise the main tasks handled in the pull request
- Create and style user update profile modal
- Format data obtained from the form to the shape required by the backend
**description of Task to be completed?**
- Connect the form to the backend by sending its data successfully
> Describe (in detail) what the task completed in the pull request does as per the relevant task

**How should this be manually tested?**
The page in question for this PR can be tested via [this link](https://supernigerians-user-dash.netlify.app/)
> Give bullet point instructions on how to setup the code and test it manually (Assume no prior experience on JS or code)
- Visit the test url above
- Click the Update profile button
- A modal appears which requests for data to update the profile

**Any background context you want to provide?**
- None
> Any pertinent information that should be considered
- None
**Screenshots (if appropriate)**

> You can have screenshots where applicable
![Screenshot from 2020-07-30 22-37-49](https://user-images.githubusercontent.com/37955249/88980588-f3d8a400-d2bb-11ea-9f3c-985253867845.png)


**Questions:**

> If something is unclear or you want some questions to be addressed by your peers, mention them here
